eclass/kde5.eclass

```diff
@@ -534,17 +535,20 @@ EOF
 				fi
 			fi
 		done
+		unset lang
```

Comment:
When `local` is used for a variable, then previous state of this variable (e.g. it being unset or its previous value if it was set) will be automatically restored after finishing of execution of given function.